std/algorithm/searching.d

```diff
@@ -1470,6 +1480,25 @@ private auto extremum(alias selector = "a < b", Range,
     assert(arr2d.extremum!"a[1]" == arr2d[1]);
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=17982
+@safe unittest
+{
+    class B
+    {
+        int val;
+        this(int val){ this.val = val; }
+    }
+
+    const(B) doStuff(const(B)[] v)
```

Comment:
This is never called

Reply:
It doesn't need to be called (it's a copy from the bug report).
Anyhow, I added an call of `doStuff` for "more coverage".